modules/ethereumNode.js

```diff
@@ -488,6 +488,22 @@ class EthereumNode extends EventEmitter {
 
                 const proc = spawn(binPath, args);
 
+                /*
+                    Assume startup succeeded after 5 seconds. At this point 
+                    IPC connections are usually possible.
+                */
+                setTimeout(() => {
```

Comment:
Agreed. I've changed it so that as soon as any data is received (stdout or stderr) it starts the timer.